torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1303,6 +1306,7 @@ def test_rref_context_debug_info(self):
         rref1 = RRef(self.rank)
 
         # don't need a barrier here as local RRef is handled by this thread
+        wait_until_pending_users_flushed()
```

Comment:
Do we need this here? As rref1 is the local OwnerRRef, there shouldn't be any user at this point?

Reply:
Yeah we shouldn't need it here. I added this before all assertions on `num_pending_users` but it's probably better to remove it from here